### PR TITLE
Fix: hcl2tojson dirs tree processing

### PR DESCRIPTION
**Pull request by ahlinc**
* Changed filenames printing to stderr by default to don't corrupt json content of writing files.
* Fixed processing of multilevel trees.
* Added skipping of unparsable files.

### CODE REVIEW

File: bin/hcl2tojson [hunk 1]
```diff
@@ -44,8 +52,15 @@ if __name__ == '__main__':
                 processed_files.add(in_file_path)
                 processed_files.add(out_file_path)
 
-                with open(in_file_path, 'r') as in_file, open(out_file_path, 'w') as out_file:
-                    print(in_file_path)
-                    json.dump(load(in_file), out_file)
+                with open(in_file_path, 'r') as in_file:
+                    print(in_file_path, file=sys.stderr, flush=True)
+                    try:
+                        t = load(in_file)
```
Comment (aoskotsky-amplify) [inline bin/hcl2tojson:58]: This probably could be named better. Maybe something like this below?

```suggestion
                        json_output = load(in_file)
```

**Comment by aoskotsky-amplify** [comment]
Thanks for this PR. Looks good except for a nitpick. Could you fix it and bump the version number so I can merge it?

File: bin/hcl2tojson (hunk 1 above)
Comment (ahlinc) [reply bin/hcl2tojson]: Thanks for the note, I've renamed it to a parsed_data variable as it actually contains python dict.

**Comment by ahlinc** [comment]
Also by a last commit I've made the OUT_PATH parameter optional to make it possible to write to stdout and use the tool's output as an input for another tools like a jq.

**Comment by aoskotsky-amplify** [comment]
Awesome thanks. Can you bump the version number here https://github.com/amplify-education/python-hcl2/blob/master/hcl2/version.py#L3 and we can get this merged

**Comment by ahlinc** [comment]
Sure. Done, bumped to 0.2.6.

**Pull request merged**